ui/raidboss/data/06-ew/trial/golbez.txt

```diff
@@ -1,6 +1,111 @@
 ### The Voidcast Dais
+#
+# -ii 844B 8463 8460 84AE 84AF 8469 8719 86CF 8473 849B 878B 84B6 84A7 84A9 8483 86B3 8484
+# -it "Golbez"
+# -ic "Gale Sphere" "Shadow Dragon" "Golbez's Shadow"
+
+# Ignored Ids:
+#  - 844B      (autos)
+#  - 8463      Terrastorm (initial cast; meteor telegraph/damage is 8465)
+#  - 8460      Duplicative Arctic Assault
+#  - 84A[EF]   Additional hits of Void Meteor buster
+#  - 8469      Subsequent Lingering Spark puddles
+#  - 8719      Duplicative Black Fang
+#  - 86CF      Duplicative Black Fang
+#  - 8473      Duplicative Black Fang
+#  - 849B      Duplicative Explosion
+#  - 878B      Duplicative Double Meteor
+#  - 84B6      Duplicative Double Meteor
+#  - 84A7      sync/vfx?
+#  - 84A9      Ground AoE telegraphs for Void Stardust
+#  - 8483      Duplicative Eventide Fall
+#  - 86B3      Duplicative Eventide Fall
+#  - 8484      sync/vfx?
+
 
 hideall "--Reset--"
 hideall "--sync--"
+hideall "--middle--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+0.0 "--sync--" sync / 104:[^:]*:1($|:)/ window 0,1
+11.0 "--middle--" sync / 1[56]:[^:]*:Golbez:84B8:/ window 12,12
+25.2 "Terrastorm" sync / 1[56]:[^:]*:Golbez:8465:/
+30.3 "Crescent Blade" sync / 1[56]:[^:]*:Golbez:846B:/
+37.0 "--middle--" sync / 1[56]:[^:]*:Golbez:84B8:/
+42.1 "Arctic Assault" sync / 1[56]:[^:]*:Golbez:845E:/
+50.3 "Crescent Blade" sync / 1[56]:[^:]*:Golbez:846B:/
+60.9 "Void Meteor" sync / 1[56]:[^:]*:Golbez:84AC:/ duration 4
+69.1 "--middle--" sync / 1[56]:[^:]*:Golbez:84B8:/
+74.2 "Lingering Spark" sync / 1[56]:[^:]*:Golbez:8467:/ duration 8
+86.5 "Crescent Blade" sync / 1[56]:[^:]*:Golbez:846B:/
+97.1 "Binding Cold" sync / 1[56]:[^:]*:Golbez:84B2:/
+106.3 "--middle--" sync / 1[56]:[^:]*:Golbez:84B8:/
+113.6 "Azdaja's Shadow" sync / 1[56]:[^:]*:Golbez:86FE:/
+113.6 "--untargetable--"
+128.6 "--targetable--"
+132.6 "Black Fang" sync / 1[56]:[^:]*:Golbez:8471:/ duration 6.5
+142.7 "--sync--" sync / 1[56]:[^:]*:Golbez:8475:/
+150.8 "--middle--" sync / 1[56]:[^:]*:Golbez:84B8:/
+156.8 "Azdaja's Shadow" sync / 1[56]:[^:]*:Golbez:8476:/
+167.1 "Shadow Crescent" sync / 1[56]:[^:]*:Golbez:8487:/
+169.4 "Rising Beacon" sync / 1[56]:[^:]*:Golbez:848F:/
+172.2 "Burning Shade" sync / 1[56]:[^:]*:Golbez:8493:/
+172.8 "--middle--" sync / 1[56]:[^:]*:Golbez:84B8:/
+186.7 "Explosion" sync / 1[56]:[^:]*:Golbez:8499:/
+191.1 "Double Meteor" sync / 1[56]:[^:]*:Golbez:84B4:/
+194.2 "--sync--" sync / 1[56]:[^:]*:Golbez:8475:/
+200.3 "--middle--" sync / 1[56]:[^:]*:Golbez:84B8:/
+205.6 "Gale Sphere" sync / 1[56]:[^:]*:Golbez:844D:/
+225.3 "--sync--" sync / 1[56]:[^:]*:Golbez:844E:/
+234.3 "Void Stardust" sync / 1[56]:[^:]*:Golbez:84A3:/
+238.2 "Void Stardust" sync / 1[56]:[^:]*:Golbez:84A5:/
+248.5 "Void Meteor" sync / 1[56]:[^:]*:Golbez:84AC:/ duration 4
```

Comment:
Is 84A3 the telegraph, 84A5 the first hit, and 84AC the continuing damage? Or, what are these ids? Mostly wondering if this needs some extra text here or an ignored id.  The extreme says `(preview)` on the telegraph, for what it's worth.

Reply:
84A3 is the initial (2.7 sec.) cast that shows the corner meteor marker.  At the conclusion of the cast, the remaining telegraphs are shown.  84A5 is a longer cast, started at the same time as 84A3, that eventually corresponds to damage from that corner meteor exploding.  84A9 (ignored) is damage from the subsequent meteors.
(84AC is Void Meteor, which is a separate mechanic -- the dual tankbuster.  Yeah, these names aren't confusing at all...)

I added (preview) on 84A3 and a small duration on 84A5 to correspond to the subsequent meteor hits.  It's a slightly longer duration than the extreme, because that's how long the 84A9 abilities take to resolve, but it otherwise matches what we're doing there.